spec/spec_helper.rb

```diff
@@ -7,7 +7,7 @@
 ActiveRecord::Base.logger = Logger.new(File.join(plugin_test_dir, "debug.log"))
 
 ActiveRecord::Base.configurations = YAML::load_file(File.join(plugin_test_dir, "db", "database.yml"))
-ActiveRecord::Base.establish_connection(ENV["DB"] || "sqlite3mem")
+ActiveRecord::Base.establish_connection((ENV["DB"] || "sqlite3").to_sym)
```

Comment:
For some reason, using the in-memory database causes the specs to fail for Rails 5.2. Because we're testing multiple versions of Rails, we can simply use sqlite3's file db storage.